apps/meteor/tests/end-to-end/api/livechat/12-priorites.ts

```diff
@@ -271,4 +277,78 @@ import { IS_EE } from '../../../e2e/config/constants';
 			expect(response.body).to.have.property('dirty', true);
 		});
 	});
+
+	describe('livechat/priority.reset', () => {
+		let priority: ILivechatPriority;
+		it('should return an "unauthorized error" when the user does not have the necessary permission', async () => {
+			await updatePermission('manage-livechat-priorities', []);
+			const response = await request.post(api('livechat/priority.reset')).set(credentials).expect(403);
+			expect(response.body).to.have.property('success', false);
+			expect(response.body).to.have.property('error');
+			expect(response.body?.error).to.contain('error-unauthorized');
+		});
+		it('should respond reset:true when a priority has been changed', async () => {
+			await updatePermission('manage-livechat-priorities', ['admin', 'livechat-manager']);
+			const {
+				body: { priorities },
+			} = await request.get(api('livechat/priorities')).set(credentials).expect('Content-Type', 'application/json').expect(200);
+
+			priority = priorities[0];
+
+			priority.name = faker.random.word();
+			const responseChange = await request
+				.put(api(`livechat/priority/${priority._id}`))
+				.set(credentials)
+				.send({
+					name: priority.name,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+			expect(responseChange.body).to.have.property('success', true);
+
+			const response = await request
+				.get(api('livechat/priority.reset'))
+				.set(credentials)
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+			expect(response.body).to.have.property('success', true);
+			expect(response.body).to.have.property('reset', true);
+		});
+		it('should reset all priorities', async () => {
+			const resetRespose = await request
+				.post(api('livechat/priority.reset'))
+				.set(credentials)
+				.send()
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+			expect(resetRespose.body).to.have.property('success', true);
+		});
```

Comment:
Add a check here to see that in fact priorities have been reset

Reply:
done as we decided in the chat